<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editar Aluno</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButtonBuscarAluno">
         <property name="text">
          <string>Buscar Aluno</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="9" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Telefone:</string>
             </property>
            </widget>
           </item>
           <item row="12" column="1">
            <widget class="QLineEdit" name="lineEditEndereco"/>
           </item>
           <item row="14" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Matrícula:</string>
             </property>
            </widget>
           </item>
           <item row="16" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Série:</string>
             </property>
            </widget>
           </item>
           <item row="15" column="0">
            <widget class="QLabel" name="labelId">
             <property name="text">
              <string>ID: </string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEditRG"/>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="lineEditCPF"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>CPF:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditNome"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Data de Nascimento</string>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Endereço:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Pai:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDateEdit" name="dateEdit"/>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Nome:</string>
             </property>
            </widget>
           </item>
           <item row="14" column="1">
            <widget class="QLineEdit" name="lineEditMatricula"/>
           </item>
           <item row="13" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="16" column="1">
            <widget class="QComboBox" name="comboBoxSerie"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>RG:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QLineEdit" name="lineEditTelefone"/>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEditPai"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Mãe:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEditMae"/>
           </item>
           <item row="17" column="0">
            <widget class="QLabel" name="labelEscola">
             <property name="text">
              <string>Escola:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Nome do Aluno: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditBuscarAlunos"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QListWidget" name="listViewAlunos">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonEditar">
         <property name="text">
          <string>Editar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonExcluir">
         <property name="text">
          <string>Excluir</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
